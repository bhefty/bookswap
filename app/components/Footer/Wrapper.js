import styled from 'styled-components'

const Wrapper = styled.footer`
  margin: 0 auto;
  text-align: center;

  .social {
    width: 30px;
    height: 30px;
  }
`

export default Wrapper
