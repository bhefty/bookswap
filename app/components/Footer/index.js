import React, { PureComponent } from 'react'

import A from 'components/A'
import Wrapper from './Wrapper'

class Footer extends PureComponent {
  render () {
    return (
      <Wrapper>
        <section>
          Made by <A href='https://billhefty.com/' target='_blank'>Bill Hefty</A>
        </section>
      </Wrapper>
    )
  }
}

export default Footer
