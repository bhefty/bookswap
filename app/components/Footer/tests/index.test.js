/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Footer from '../index'

describe('<Footer />', () => {
  it('should render the page', () => {
    const tree = renderer.create(
      <Footer />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
