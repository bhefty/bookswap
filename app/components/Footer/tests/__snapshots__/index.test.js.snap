// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render the page 1`] = `
<footer>
  <h1>
    Just a footer...
  </h1>
</footer>
`;
